windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -775,6 +779,30 @@ procedure TMergeKeyboardInfo.CheckOrAddMinKeymanVersion;
     json.AddPair('minKeymanVersion', MinVersionString);
 end;
 
+//
+// Add helpLink, from commandline parameter
+//
+procedure TMergeKeyboardInfo.AddHelpLink;
+var
+  v: TJSONValue;
+
+begin
+  // Validate pre-existing helpLink
+  v := json.GetValue('helpLink');
+  if v <> nil then
+  begin
+    if v.Value.IndexOf('https://help.keyman.com/keyboard/') < 0 then
```

Comment:
This should be `if ... <> 0`

Reply:
I want this to raise when `IndexOf()` returns -1 because the string isn't found,